pkgs/development/python-modules/jc/default.nix

```diff
@@ -5,22 +5,25 @@
 , xmltodict
 , pygments
 , isPy27
+, pytestCheckHook
 }:
 
 buildPythonPackage rec {
   pname = "jc";
-  version = "1.14.0";
+  version = "1.14.1";
   disabled = isPy27;
 
   src = fetchFromGitHub {
     owner = "kellyjonbrazil";
     repo = "jc";
     rev = "v${version}";
-    sha256 = "0js3mqp6xxg45qsz8wnyyqf4m0wj1kz67bkmvirhdy7s01zhd5hq";
+    sha256 = "1vzzz7dlg6apxhcl0qkfdpp2v9d0q6jyafpfmklkcbjs31zvwcsw";
   };
 
   propagatedBuildInputs = [ ruamel_yaml xmltodict pygments ];
 
+  checkInputs = [ pytestCheckHook ];
```

Comment:
Now running tests with pytestCheckHook because jc generates wrong unix
timestamps in test_last_F_ubuntu_20_4 test_last_wF_centos_7_7 otherwise.